pkg/rgrpc/rgrpc.go

```diff
@@ -292,6 +293,7 @@ func (s *Server) getInterceptors(unprotected []string) ([]grpc.ServerOption, err
 		token.NewUnary(),
 		log.NewUnary(),
 		recovery.NewUnary(),
+		useragent.NewUnary(),
```

Comment:
Move this above the log unary. I tried to move it above that but I still see the `grpc-go` agent being logged
`2021-11-04 14:00:55.491 DBG ../../internal/grpc/interceptors/log/log.go:69 > unary code=OK end="04/Nov/2021:14:00:55 +0100" from=tcp://127.0.0.1:52341 pid=46490 pkg=rgrpc start="04/Nov/2021:14:00:55 +0100" time_ns=2803304 uri=/cs3.storage.provider.v1beta1.ProviderAPI/ListContainer user-agent=grpc-go/1.26.0`